### ISSUE TITLE
iiwa_allegro.yml

### ISSUE THREAD

**Issue by xianjir** [tylerlum/human2sim2robot#8]
Where could I find this "iiwa_allegro.yml"? And what else should I deploy if I want to use the other hardware like Franka except alternating this yml?

**Comment by yys-abu** [comment]
> Where could I find this "iiwa_allegro.yml"? And what else should I deploy if I want to use the other hardware like Franka except alternating this yml?

Have you successfully configured your environment? Why can't I dowmload https://gitlab.com/tylerlum/fabrics-sim.git. It is Invalid.

**Comment by xianjir** [comment]
> > Where could I find this "iiwa_allegro.yml"? And what else should I deploy if I want to use the other hardware like Franka except alternating this yml?
> 
> Have you successfully configured your environment? Why can't I dowmload https://gitlab.com/tylerlum/fabrics-sim.git. It is Invalid.

I can't import the fabrics either because it haven't been open-souced yet, but the author said we could use pytorch_kinematics to replace it. But I don't know how.
https://github.com/tylerlum/human2sim2robot/issues/4#issuecomment-3041742497

**Comment by yys-abu** [comment]
> > > 我在哪里可以找到这个“iiwa_allegro.yml”？如果我想使用其他硬件，比如 Franka，除了交替使用这个 yml，我还应该部署什么？
> > 
> > 
> > 是否已成功配置环境？为什么我不能下载 https://gitlab.com/tylerlum/fabrics-sim.git。它是无效的。
> 
> 我也不能进口面料，因为它还没有开放，但老板说我们可以用pytorch_kinematics来更换它。但我不知道怎么做。 [#4](https://github.com/tylerlum/human2sim2robot/issues/4) 顺便说一句，你有没有配置过hamer_depth并成功运行它？我遇到了一个接地恐龙问题还没有解决。 [#3](https://github.com/tylerlum/human2sim2robot/issues/3)
I successfully ran it using this configuration:
absl-py                   2.3.1
addict                    2.4.0
aiohappyeyeballs          2.6.1
aiohttp                   3.12.14
aiosignal                 1.4.0
antlr4-python3-runtime    4.9.3
asttokens                 3.0.0
async-timeout             5.0.1
attrs                     25.3.0
beautifulsoup4            4.13.4
black                     25.1.0
blinker                   1.9.0
braceexpand               0.1.7
certifi                   2022.12.7
charset-normalizer        2.1.1
chumpy                    0.71               /home/abu/chumpy-master
click                     8.2.1
cloudpickle               3.1.1
cmake                     3.25.0
comm                      0.2.2
ConfigArgParse            1.7.1
contourpy                 1.3.2
cycler                    0.12.1
Cython                    3.1.2
dash                      3.1.1
decorator                 5.2.1
detectron2                0.6                /home/abu/detectron2-main
docstring_parser          0.17.0
einops                    0.8.1
exceptiongroup            1.3.0
executing                 2.2.0
fastjsonschema            2.21.1
filelock                  3.13.1
Flask                     3.1.1
fonttools                 4.59.0
freetype-py               2.5.1
frozenlist                1.7.0
fsspec                    2025.7.0
fvcore                    0.1.5.post20221221
gdown                     5.2.0
grpcio                    1.73.1
hamer                     0.0.0              /home/abu/hamer
hamer_depth               0.1.0              /home/abu/hamer_depth
hf-xet                    1.1.5
huggingface-hub           0.33.4
hydra-core                1.3.2
idna                      3.4
imageio                   2.37.0
importlib_metadata        8.7.0
iopath                    0.1.9
ipython                   8.37.0
ipywidgets                8.1.7
itsdangerous              2.2.0
jedi                      0.19.2
Jinja2                    3.1.4
joblib                    1.5.1
json-tricks               3.17.3
jsonschema                4.25.0
jsonschema-specifications 2025.4.1
jupyter_core              5.8.1
jupyterlab_widgets        3.0.15
kiwisolver                1.4.8
lazy_loader               0.4
lightning-utilities       0.14.3
lit                       15.0.7
Markdown                  3.8.2
markdown-it-py            3.0.0
MarkupSafe                2.1.5
matplotlib                3.10.3
matplotlib-inline         0.1.7
mdurl                     0.1.2
mmcv                      1.3.9
mmdet                     3.3.0
mmengine                  0.10.7
mmpose                    0.24.0             /home/abu/hamer/third-party/ViTPose
mpmath                    1.3.0
multidict                 6.6.3
munkres                   1.1.4
mypy_extensions           1.1.0
narwhals                  1.47.1
nbformat                  5.10.4
nest-asyncio              1.6.0
networkx                  3.3
numpy                     1.23.5
omegaconf                 2.3.0
open3d                    0.19.0
opencv-python             4.12.0.88
packaging                 25.0
pandas                    2.3.1
parso                     0.8.4
pathspec                  0.12.1
pexpect                   4.9.0
pillow                    11.3.0
pip                       25.1
platformdirs              4.3.8
plotly                    6.2.0
portalocker               3.2.0
prompt_toolkit            3.0.51
propcache                 0.3.2
protobuf                  6.31.1
ptyprocess                0.7.0
pure_eval                 0.2.3
pycocotools               2.0.10
pyglet                    2.1.6
Pygments                  2.19.2
PyOpenGL                  3.1.0
pyparsing                 3.2.3
pyquaternion              0.9.9
pyrender                  0.1.45
PySocks                   1.7.1
python-dateutil           2.9.0.post0
pytorch-lightning         2.0.0
pytz                      2025.2
PyYAML                    6.0.2
referencing               0.36.2
regex                     2024.11.6
requests                  2.28.1
retrying                  1.4.1
rich                      14.0.0
rpds-py                   0.26.0
rtree                     1.4.0
safetensors               0.5.3
scikit-image              0.25.2
scikit-learn              1.7.1
scipy                     1.15.3
setuptools                80.9.0
shapely                   2.1.1
shtab                     1.7.2
six                       1.17.0
smplx                     0.1.28
soupsieve                 2.7
stack-data                0.6.3
sympy                     1.13.3
tabulate                  0.9.0
tensorboard               2.20.0
tensorboard-data-server   0.7.2
termcolor                 3.1.0
terminaltables            3.1.10
threadpoolctl             3.6.0
tifffile                  2025.5.10
timm                      1.0.17
tokenizers                0.19.1
tomli                     2.2.1
torch                     2.0.1+cu117
torchmetrics              1.7.4
torchvision               0.15.2+cu117
tqdm                      4.67.1
traitlets                 5.14.3
transformers              4.44.2
trimesh                   4.7.1
triton                    2.0.0
typeguard                 4.4.4
typing_extensions         4.14.1
tyro                      0.9.26
tzdata                    2025.2
urllib3                   1.26.13
wcwidth                   0.2.13
webdataset                1.0.2
Werkzeug                  3.1.3
wheel                     0.45.1
widgetsnbextension        4.0.14
xtcocotools               1.14.3
yacs                      0.1.8
yapf                      0.43.0
yarl                      1.20.1
zipp                      3.23.0
